app/javascript/components/pages/Home.js

```diff
@@ -2,32 +2,26 @@ import React from "react"
 import { useNavigate, NavItem, Nav } from "react-router-dom"
 
 const Home = ({ current_user, logged_in }) => {
-
   if (logged_in) {
```

Comment:
while this works, you can also use the 
`{logged_in && ( 
   ...code here ...
)}`
syntax

Reply:
Okay cool, Thank you!